src/Test/LicTest.java

```diff
@@ -63,6 +63,51 @@ void licFive() {
 
     @Test
     void licSix() {
+        //all points same distance away
+        Point2D.Double[] points = {
+                new Point2D.Double(0, 0 ),
+                new Point2D.Double(0,0),
+                new Point2D.Double(0, 0),
+                new Point2D.Double(0,0),
+                new Point2D.Double(0, 0)
+        };
+        params.setPoints(points);
+        assertFalse(Lic.licSix(params));
+
+        //First and last point the same, one point further than dist away from the coincident point
+        Point2D.Double[] points2 = {
+                new Point2D.Double(0, 0 ),
+                new Point2D.Double(0,0),
+                new Point2D.Double(0, params.getDist()*2),
```

Comment:
the points on line 81 and 82 should switch places for this test, so that the outlier point is the one immediately preceding the last point. This asserts that licSix doesn't end the inner for-loop too early
```suggestion
                new Point2D.Double(0, 0),
```